src/shogun/kernel/WeightedDegreeRBFKernel.h

```diff
@@ -77,6 +77,18 @@ class CWeightedDegreeRBFKernel: public CDotKernel
 		virtual EFeatureType get_feature_type() { return F_DREAL; }
 
 
+		/** Set width
+		 *
+		 * @param w new width
+		 */
+		void set_width(float64_t w) { width=w; }
```

Comment:
we should actually assert positivity here, but since this will change in the future, nevermind :)